lib/sheltertech/db/fixture_populator.rb

```diff
@@ -580,6 +594,12 @@ module Constants # rubocop:disable Metrics/ModuleLength
         'Low-Income'
       ].freeze
 
+      PARENT_ELIGIBILITY_NAMES = [
+        'Foster Youth',
```

Comment:
We might want to consider adding data that mimics UCSF's structure in a later PR.